docs/src/tutorials/performance.jl

```diff
@@ -8,12 +8,12 @@
 # * Identify problematic kernel invocations: you may be launching thousands of kernels which could be fused into a single call;
 # * Find stalls, where the CPU isn't submitting work fast enough to keep the GPU busy.
 
-# If that isn't sufficient, and you identified a kernel that executes slowly, you can try using NSight Compute to analyze that kernel in detail. Some things to look out for in order of importance:
-# * Memory optimizations are the most important area for performance. Hence optimizing memory accesses, e.g., avoiding needless global accesses (buffering in shared memory instead) and coalescing accesses can lead to big performance improvements;
-# * Launching more threads on each streaming multiprocessor can be acheived by lowering register pressure and reducing shared memory usage, the tips below outline the various ways in which register pressure can be reduced;
-# * Using Float32's instead of Float64's can provide significantly better performance;
-# * Avoid using control flow instructions such as `if` which cause branches, e.g. replace an `if` with an `ifelse` if possible;
-# * Increase the arithmetic intensity in order for the GPU to be able to hide the latency of memory accesses.
+# If that isn't sufficient, and you identified a kernel that executes slowly, you can try using NSight Compute to analyze that kernel in detail. Some things to try in order of importance:
+# * Optimizing memory accesses, e.g. avoiding needless global accesses (buffering in shared memory instead) or coalescing accesses;
+# * Launching more threads on each streaming multiprocessor can be achieved by lowering register pressure and reducing shared memory usage, the tips below outline the various ways in which register pressure can be reduced;
+# * Using Float32's instead of Float64's;
+# * Avoiding the use of control flow instructions such as `if`s which cause branches, e.g. try replacing an `if` with an `ifelse`;
+# * Increasing the arithmetic intensity in order for the GPU to be able to hide the latency of memory accesses.
```

Comment:
I think I like the imperative more than use of the passive voice... But I'm not a native speaker, so feel free to ignore.

In any case, now that I read this, maybe we should specify that control flow isn't bad per se, only when the branches diverge within a warp. So maybe:

> Avoid the use of control flow that diverges within a branch, i.e., make sure `while` or `for` loops behave identically across the entire warp, replace `if` with `ifelse` if you need to branch within a warp;